README

```diff
@@ -91,9 +102,3 @@ if you want to fix them.
 
 Archive file names are not re-encoded to current locale. You can use `iconv`
 for that.
-
-Archive file names are truncated to 100 characters (the limit for original tar
```

Comment:
As far as I understand, there are 3 incompatible tar extensions that handle long file names. This is one of the reasons I didn't bother to implement long file names in the first place ;-) Anyway, please document here that this code works with GNU-style long file names, but not POSIX-style or old-GNU.